### ISSUE TITLE
is ScratchX editor really 434?

### ISSUE THREAD

**Issue by kaschm** [LLK/scratchx#14]
This may not be an issue, but ScratchX editor says 434. Is it really running this version? (Scratch production is 435+)

![screen shot 2015-05-11 at 10 29 47 am](https://cloud.githubusercontent.com/assets/782228/7567094/e57f281c-f7c8-11e4-8deb-a3ba0585b5ad.png)

**Comment by cwillisf** [comment]
Yep, it's really based on 434... for now. Eventually (hopefully soon) I will do some merging and from then on ScratchX should stay roughly in sync with the main site.
